public/achievementinspector.php

```diff
@@ -177,7 +177,7 @@ function toggleAllCodeRows() {
             HTML;
         }
 
-        if ($fullModifyOK) {
+        if ($partialModifyOK || $fullModifyOK) {
```

Comment:
I can't help but notice there is an identical conditional for another HTML block just above this.

Let's consolidate those conditionals into one:

```php
if ($partialModifyOK || $fullModifyOK) {
    echo <<<HTML
        <p align="justify">...</p>

        <p align="justify">...</p>
    HTML;
}
```

Reply:
Instructions combined and view conditional consolidated